src/features/surveys/repos/SurveysRepo.ts

```diff
@@ -45,8 +45,9 @@ type ZetkinSurveyOptionsQuestionElementPostBody = {
   type: ELEMENT_TYPE.QUESTION;
 };
 
-type ZetkinSurveyElementPatchBody =
+export type ZetkinSurveyElementPatchBody =
   | ZetkinSurveyTextElementPatchBody
+  | Omit<ZetkinTextQuestion, 'required'>
   | Partial<Omit<ZetkinSurveyOptionsQuestionElementPostBody, 'type'>>
```

Comment:
I'm not sure I understand here - if a fourth type is needed here my thought is that the type on the last line,   | `Partial<Omit<ZetkinSurveyTextQuestionElementPostBody, 'type'>>` does not fill its purpose and should be changed? In my head it should be enough to have a union of three types her, but I am not sure.